core/cli/src/main/java/mil/nga/giat/geowave/core/cli/operations/config/SetCommand.java

```diff
@@ -46,44 +43,68 @@
 	@Parameter(names = {
 		"--password"
 	}, description = "boolean (true|false) - specify if the value being set is a password and should be encrypted in the configurations")
-	private String password = null;
+	private final String password = null;
 
 	private boolean isPassword;
 
 	@Override
-	public boolean prepare(
-			OperationParams params ) {
-		super.prepare(params);
-		if (password != null && !"".equals(password.trim())) {
+	public void execute(
+			final OperationParams params ) {
+		if ((password != null) && !"".equals(password.trim())) {
 			isPassword = Boolean.parseBoolean(password.trim());
 		}
-		return true;
+		computeResults(params);
 	}
 
+	/**
+	 * Add rest endpoint for the set command. Looks for GET params with keys
+	 * 'key' and 'value' to set.
+	 *
+	 * @return string containing json with details of success or failure of the
+	 *         set
+	 */
 	@Override
-	public void execute(
-			OperationParams params ) {
-
-		Properties existingProps = getGeoWaveConfigProperties(params);
+	public Object computeResults(
+			final OperationParams params ) {
+		try {
+			return setKeyValue(params);
+		}
+		catch (WritePropertiesException | ParameterException e) {
+			// TODO GEOWAVE-rest-project server error status message
```

Comment:
can't we throw an exception or does returning null set error status?

Reply:
done

Reply:
can we log these?